client/src/components/generic/ExecImage/ExecImage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import Image from "next/image"
+// import "./ExecImage.css"
+
+export interface ExecImageProps {
+  src: string
+  alt: string
+  title: string
+  name: string
+}
+
+const ExecImage: React.FC<ExecImageProps> = ({ src, alt, title, name }) => {
+  return (
+    <div className="group relative flex h-[163px] w-[163px] flex-shrink-0 overflow-hidden rounded-[4px]">
+      <Image
+        src={src}
+        alt={alt}
+        width={163}
+        height={163}
+        className="h-full w-full object-cover transition-opacity duration-300"
+      />
+      <div className="absolute left-0 top-0 h-full w-full bg-[rgba(40,61,135,0.8)] opacity-0 transition-opacity duration-300 group-hover:opacity-100"></div>
```

Comment:
nit: 
```suggestion
      <div className="absolute left-0 top-0 h-full w-full bg-dark-blue-100 opacity-0 transition-opacity duration-300 group-hover:opacity-100"></div>
```

check out `client\tailwind.config.ts` for preset colours

Reply:
How do i make it 0.8 opacity

Reply:
Suggest getting tailwind intellisense, there should be `opacity-80`